eng/common/scripts/Verify-Links.ps1

```diff
@@ -69,16 +69,18 @@ function ResolveUri ([System.Uri]$referralUri, [string]$link)
   $linkUri = [System.Uri]$link;
   # Our link guidelines do not allow relative links so only resolve them when we are not
   # validating links against our link guidelines (i.e. !$checkLinkGuideance)
-  if(!$checkLinkGuidance) {
-    if (!$linkUri.IsAbsoluteUri) {
-    # For rooted paths resolve from the baseUrl
-      if ($link.StartsWith("/")) {
-        Write-Verbose "rooturl = $rootUrl"
-        $linkUri = new-object System.Uri([System.Uri]$rootUrl, ".$link");
-      }
-      else {
-        $linkUri = new-object System.Uri($referralUri, $link);
-      }
+  if ($checkLinkGuidance -and !$linkUri.IsAbsoluteUri) {
+    return $linkUri
+  }
+
+  # For rooted paths resolve from the baseUrl
```

Comment:
This comment is only for the if block below. The else block is used to resolve non-rooted relative paths.